frameworks/autosklearn/setup.sh

```diff
@@ -15,6 +15,8 @@ if [[ -x "$(command -v apt-get)" ]]; then
     SUDO apt-get install -y build-essential swig
 fi
 
+PIP install packaging
```

Comment:
by convention we tend to put all those additional dependencies in a `requirements.txt` file, it's easier to see what is needed.
```
PIP install -r $HERE/requirements.txt
```